cypress/e2e/Courses/Certificates/StandardPlanUnawardedCourseCertificate.feature

```diff
@@ -1,14 +1,16 @@
-## when FF MILESTONE_PLANS_UPDATES_PART_ONE_ENABLED is activate please remove .hold extension from this file
+@course @certificate @free-plan @business:high-impact @low-likely
```

Comment:
Low likely because we don't touch this every day, and almost no user is reaching the credit limit.